helm-chart/taweret/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: taweret
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
I think we can remove all the comments here, they look like a guide for the first time this file is setup.

Reply:
yep, removed